subiquity/server/controllers/filesystem.py

```diff
@@ -157,13 +158,8 @@ def is_valid(self) -> bool:
     def capability_info_for_gap(
         self,
         gap: gaps.Gap,
+        install_min: int,
```

Comment:
This is fine for now, but it strikes me that the only use this methods makes of its arguments now is "gap.size < install_min", which I don't know feels a bit strange, like maybe callers should be checking that and there should be a method on CapabilityInfo that just returns a "too small" variant of the info. Or something.